corehq/apps/hqadmin/static/hqadmin/js/admin_restore.js

```diff
@@ -1,6 +1,21 @@
-/* globals hqDefine */
+/* globals hqDefine ace */
 hqDefine('hqadmin/js/admin_restore', function () {
     $(function() {
         $("#timingTable").treetable();
+        var element = document.getElementById("payload");
+        var editor = ace.edit(
```

Comment:
How much are these options going to vary on different pages? Is it too soon to make a standard set of options?

Reply:
these are totally different to the other use, so "a lot".